src/test/rustdoc/deref-const-fn.rs

```diff
@@ -0,0 +1,23 @@
+#![crate_name = "foo"]
+
+// @has 'foo/struct.Bar.html'
+pub struct Bar;
+
+impl Bar {
+    // @has - '//*[@id="method.len"]' 'pub const fn len(&self) -> usize'
+    pub const fn len(&self) -> usize { 0 }
```

Comment:
Can you test const (in)stability in this test as well? (Using `#![feature(staged_api)]`.)

Reply:
And can you also add a comment to the top of this test explaining what it tests?

Reply:
> Can you test const (in)stability in this test as well? (Using `#![feature(staged_api)]`.)

I don't think rustdoc handles `~const` yet, so not sure how useful it would be (also there are a lot of things from https://internals.rust-lang.org/t/pre-rfc-revamped-const-trait-impl-aka-rfc-2632/15192...).

Reply:
What do you mean? I'm talking about

```rust
#[rustc_const_stable(...)]
```

which is totally unrelated to `~const` (aside from them both being related to const evaluation).

Reply:
Oh sorry. Yes, you're absolutely right! I'll extend the test.